src/MQTTClient.php

```diff
@@ -645,7 +644,7 @@ public function loop(bool $allowSleep = true, bool $exitWhenQueuesEmpty = false,
                         case 12:
                             $this->handlePingRequest();
                             break;
-                        case 13;
+                        case 13:
```

Comment:
I really wonder how this didn't trigger a compile error and why PhpStorm wasn't shouting at me for doing this (yes, I make sure PhpStorm reports a green checkmark for code style and issues ...).

Reply:
It has the exact same semantic in PHP.. still hard to watch.